src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -92,15 +93,18 @@ class SaveInProgressIntro extends React.Component {
         const lastUpdated =
           savedForm.lastUpdated || _.get('metadata.lastUpdated', savedForm);
         const savedAt = this.props.lastSavedDate
-          ? moment(this.props.lastSavedDate)
-          : moment.unix(lastUpdated);
-        const expiresAt = moment.unix(savedForm.metadata.expiresAt);
-        const expirationDate = expiresAt.format('MMMM D, YYYY');
-        const isExpired = expiresAt.isBefore();
+          ? new Date(this.props.lastSavedDate)
+          : fromUnixTime(lastUpdated);
```

Comment:
Cypress tests are failing because `lastUpdated` is undefined and `date-fns` is not as tolerant as moment.js. I'll look at refactoring this tomorrow.